warrior/Actions/SnmpActions/common_snmp_actions.py

```diff
@@ -681,7 +681,7 @@ def verify_snmp_action(self, system_name, snmp_result, mib_string=None
                     testcase_Utils.pNote("No SNMP Result Present!", 'error')
         for element in result_list:
             if mib_string:
-                if mib_string in element[0] and snmp_result in element[-1]:
+                if re.search(mib_string, element[0]) and re.search(snmp_result, element[-1]):
```

Comment:
Add this regex search support in below else block as well(line no. 690).

Reply:
Edited docstring and put the regex in else block, WAR-1560 is the ticket for pylint